the-journal-of-foot-and-ankle-surgery.csl

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="minimal" default-locale="en-US">
+  <info>
+    <title>The Journal of Foot & Ankle Surgery</title>
```

Comment:
Ah ok. The validator didn't let me check it. When I copied the code and click on validate it looks like I only copied the first few lines.